components/browser/errorpages/src/main/assets/parseQueryAndInject.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handles the parsing of the ErrorPages URI and then passes them to injectValues
+ */
+window.parseQuery = function(queryString) {
```

Comment:
Why does this need to be global?

Reply:
Oops, thought I needed that to access it in `window.onLoad` but I guess not!